simmechanics/data/icub3/ICUB_3_all_options_gazebo.yaml.in

```diff
@@ -832,4 +832,60 @@ XMLBlobs:
             <gazebo reference="r_wrist_yaw">
               <implicitSpringDamper>1</implicitSpringDamper>
             </gazebo>
+    - |
+            <gazebo reference="r_upper_arm">
+              <selfCollide>true</selfCollide>
+              <collision>
+                <origin xyz="0.030743125543818514 0.141531075879301 0.06954757274337188" rpy="-0.0715644603308 0.0619699233851 -1.84623504024"/>

```

Comment:
The duplication problem seems still here?

Reply:
I didn't check the gazebo options - I have now removed them and committed the changes!